src/containers/App/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import logo from '../../styles/svgs/logo.svg';
+import appSchema from '../../validations/app';
+import useForm from 'react-hook-form';
```

Comment:
What's the advantage of this over formik?

Reply:
It works faster, it's easier to read, it's based on React hooks, prevents unnecessary re-renders

Reply:
As far as I know formik uses hooks under the hood as well

Reply:
Also the question is if it gives us all the features we want

Reply:
But Formik re-renders much more then React hook form.
Yes, we have all the features we'll need.

Reply:
Let's have a good discussion around this, @ao-armen has found some interesting stuff. Armen was looking at this, seems pretty intereseting

https://react-hook-form.com/